projects/common/src/lib/controls/refresh-select/refresh-select.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, EventEmitter, Input, OnChanges, OnInit, Output } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { IoTEnsembleState } from '../../state/iot-ensemble.state';
+
+@Component({
+  selector: 'lcu-refresh-select',
+  templateUrl: './refresh-select.component.html',
+  styleUrls: ['./refresh-select.component.scss']
+})
+export class RefreshSelectComponent implements OnInit, OnChanges {
+
```

Comment:
Create 
// Fields, Properties, Constructors, Life Cycle, API Methods, Helpers
comment section headers like the other components.